### ISSUE TITLE
Add the ability to add aria-label to ChoiceList

### ISSUE THREAD

**Issue by SparrowSpit** [CMSgov/design-system#708]
**Describe the issue:**
We (MCT) would like the ability to add an `aria-label` to the ChoiceList component. One of our radio input groups requires an `aria-label` to clarify for screen reader users the text displayed in the `legend`. 

https://design.cms.gov/components/choice/

**Comment by nichia** [comment]
Hey @SparrowSpit, can you provide an example of the `aria-label` usage.

Tracked here: https://jira.cms.gov/browse/WNMGDS-487

**Comment by bernardwang** [comment]
To clarify, do you need the `aria-label` on an individual radio button? Or do you need the label on the `legend` in the `ChoiceList` component?

**Comment by nichia** [comment]
@davidakennedy - can you help confirm whether radio buttons wrapped in `fieldset` is accessible by screen readers? Specifically, the screen reader should be reading out the text displayed in the `legend` without adding additional aria-label?

**Comment by nichia** [comment]
Or - if aria-labels attributes are needed for screen readers, which element should they be placed with?

**Comment by davidakennedy** [comment]
Hey @nichia, thanks for asking! I talked with Dan Bivins, a designer on MCT to get more context behind the need for the change here.

Here's a mockup of the screen that prompted the issue:

<img width="689" alt="" src="https://user-images.githubusercontent.com/1473618/83800981-54f64980-a676-11ea-8561-dbd6d16603b7.png">

Because the text in the legends are in all caps, some screen readers announce them by spelling them out, which can be confusing for screen reader users. In Dan's testing, both the words `AGE` and `SEX` were problematic. Adding an `aria-label` to the legend element would be the best route here. That allows the label to override the visual text in the legend.

I would keep the property optional in the React component because most of the time, it won't be needed.

Screen readers often read the legend element when browsing radio buttons and checkboxes, if it's used. It gives those elements more context.

**Comment by nichia** [comment]
Thanks for clarifying @davidakennedy - we'll add an optional property `legendAriaLabel` to <ChoiceList> component. This optional `aria-label` will be passed to the <FormLabel> component  and get added to the legend element.

**Comment by line47** [comment]
Additional properties can be added to the <FormLabel> component like `aria-label` which in turn can be used to fix the issue mentioned above. However, because this is an isolated case we recommend instead of using a the <ChoiceList> component that you use the <FormLabel> and <Choice> components to achieve adding an `aria-label`. 

```JSX
<fieldset>
       <FormLabel component="legend" aria-label="testing Aria">
           Legend text   
       </FormLabel>
       <Choice defaultChecked hint="Checkbox A hint" name="checkbox_choice" type="checkbox" value="a">
           Checkbox A
       </Choice>
       <Choice name="checkbox_choice" type="checkbox" value="b">
           Checkbox B
       </Choice>
</fieldset>
```

**Comment by line47** [comment]
@SparrowSpit - let me know if you have questions about implementing this approach
